packages/xhr-mock/src/MockXMLHttpRequest.ts

```diff
@@ -193,7 +195,17 @@ export default class MockXMLHttpRequest extends MockXMLHttpRequestEventTarget
   }
 
   get responseXML(): Document | null {
-    throw notImplementedError;
+    if (this.responseType === '' || this.responseType === 'document') {
+      if (
+        this.readyState === this.DONE &&
+        this.res.body() instanceof Document
```

Comment:
Can we please check `Document` is defined before using it - in some environments e.g. NodeJS it won't be defined 🙏

Reply:
Done, sorry for the delay ....

Reply:
No worries, same!